en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc

```diff
@@ -0,0 +1,31 @@
+= MuleSoft Automation Package 
+
+The https://www.mulesoft.com/automation-pricing-new[MuleSoft Automation Package] enables you to log in to Anypoint Platform to access MuleSoft Composer and xref:rpa-home::index.adoc[MuleSoft Robotic Process Automation (RPA)]. With the Hyperautomation Package, MuleSoft Composer now has everything you need to connect apps and create automations.  
```

Comment:
By reading this article, it is not clear to me if the Automation and Hyperautomation are two different Packages with different features, or just the same thing with different names.